src/vm/internal/otable.cc

```diff
@@ -92,9 +92,7 @@ int ObjectTable::showStatus(outbuffer_t* out, int verbose) {
     default:
       break;
   }
-  std::string content;
-  ss >> content;
-  outbuf_add(out, content.c_str());
+  outbuf_add(out, ss.str().c_str());
```

Comment:
stringstream.str().c_str() will return an (invalid) pointer to an deallocated string ( because expression is finished), and it is an common misuse. So this won’t work.

Which is why I changed it in the first place.

Reply:
we need to do 
```
{
  std::string content = ss.str()
  outbuf_add(out, content.c_str());
}
```

Reply:
Oops...
Missed that note...